baiji/s3.py

```diff
@@ -880,27 +894,56 @@ def _get_etag(self, netloc, remote_path):
             raise KeyNotFound("s3://%s/%s not found on s3" % (netloc, remote_path))
         return k.etag.strip("\"") # because s3 seriously gives the md5sum back wrapped in an extra set of double quotes...
 
+    def _build_etag(self, local_path, n_parts, part_size):
```

Comment:
Ah, I see this was a copy and paste from below. Still, I think it would help to have a little high-level context.